dlp/metadata.py

```diff
@@ -21,11 +21,12 @@
 
 
 # [START list_info_types]
```

Comment:
This should be `dlp_list_info_types`, unless Python doesn't use region tags. (The lack of region tags may confuse @jmdobry's sample-tracking tools,  but I'm not sure.)